shared/src/commonMain/kotlin/io.projectnewm.shared/login/models/LoginResponse.kt

```diff
@@ -0,0 +1,35 @@
+package io.projectnewm.shared.login.models
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class LoginResponse(
+    @SerialName("accessToken")
+    val accessToken: String,
+    @SerialName("refreshToken")
+    val refreshToken: String
+)
+
+sealed class LoginStatus {
```

Comment:
Is there a way we can just throw these errors?  Or if we decide to share view models, we can just set the error there and the client view's can show them.